ostorlab.yaml

```diff
@@ -69,3 +69,7 @@ args:
     type: "string"
     description: "Template of timing settings (T0, T1, ... T5)."
     value: "T4"
+  - name: "script"
```

Comment:
Let's call it scripts and pass a list of strings.
You persist all the scripts in one folder and you pass the folder path to nmap

Reply:
+1